.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+
+---
+
+### Please read the following before submitting:
+- Please read our Code of Conduct: https://github.com/KuriusMTL/HackItForward/blob/master/CODE_OF_CONDUCT.md
```

Comment:
```suggestion
- Please read our [Code of Conduct](https://github.com/KuriusMTL/HackItForward/blob/master/CODE_OF_CONDUCT.md).
```

Reply:
I realized this is not displayed in markdown in issues, so we should keep the semi-colon for readability. Same thing for the bug template we have right now. I looked at other repos and this is what they do as well

Reply:
Fair enough, feel free to add this one + fix the other one in a single separate PR.